src/libServer/IsolatedServer.h

```diff
@@ -49,6 +50,24 @@ class IsolatedServer : public LookupServer,
     response = this->CreateTransaction(request[0u]);
   }
 
+  inline virtual void GetEthSendRawTransactionI(const Json::Value& request,
+                                                Json::Value& response) {
+    (void)request;
```

Comment:
If we don't use request, should we enclose it with / * */ in the arg list?

Reply:
Oh. Request is used. The (void) was a copy/paste error in this instance. I don't mind refactoring to use /* */ instead of a void cast in a future PR if you like?